libs/core/src/schemas/entities.schemas.ts

```diff
@@ -327,10 +327,10 @@ export const ContestManager = z
       commonProtocol.WeiDecimals[commonProtocol.Denominations.ETH],
     ),
     created_at: z.date(),
-    paused: z
+    cancelled: z
```

Comment:
not related to this PR but I am curious => if the contest ended, would `cancelled` flip to `true`?